### PR TITLE
Added "unattest" functionality for firmatecknare

### PR DESCRIPTION
**Pull request by TerraDOOM**
Added functionality so that people with attest-firmatecknare may unattest expenses, at popular request (Felix Murnion)

### CODE REVIEW

**Review by DouglasFischer — DISMISSED** [review]
The action should also send a message on the receipt, stating what has been done.

**Comment by Adasjo** [comment]
Should this really be a feature? The main reason for an "attest" is to approve an expense or something. There should instead be a block that we can't do a payout for the expense and firmatecknare gives a valid reason. As @DouglasFischer mentioned, a message will be sent to the expense owner and the one who did the "attest".

File: admin/views.py [hunk 1]
```diff
@@ -67,6 +67,23 @@ def attest_expense_part(request, pk):
...
+        if not request.user.profile.is_admin():
+            return HttpResponseRedirect(reverse('expenses-show', kwargs={'pk': int(pk)}))
+
+        for part in expense_parts:
+            part.unattest(request.user)
+    except ObjectDoesNotExist:
+        raise Http404("Kvittodelarna finns inte")
+
+    return HttpResponseRedirect(reverse('expenses-show', kwargs={'pk': int(pk)}))
+
```
Comment (foodelevator) [inline admin/views.py]: This doesn't check that expense hasn't been reimbursed, which could lead to a reimbursed payment being unattested if the expense was loaded before the reimbursement, so the button would still be there.

**Review by foodelevator — CHANGES_REQUESTED** [review]
I felt like joining in on the request changes train

**Review by foodelevator — APPROVED** [review]
You should unattest a part of an expense, not the whole expense, but good enough i guess

**Comment by foodelevator** [comment]
Vafan vill den inte rebasea för?

**Pull request merged**